README.md

```diff
@@ -14,24 +17,50 @@ Ruuvi Firmware version 3. Built on top of Nordic SDK 15, uses both Ruuvi and ext
 Under development, please follow [Ruuvi Blog](https://blog.ruuvi.com) for details. The project is in beta stage, no breaking changes are intented but will be done if absolutely necessary for some reason.  
 
 # Setting up
+
+## Prerequisites
+* gcc
+* [gcc-arm-none-eabi-7-2018-q2-update](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads/7-2018-q2-update) CHeck that nRF5_SDK_15.3.0_59ac345/components/toolchain/gcc  Makefile.posix or Makefile.windows points to it.
+* git
+* on Mac OS xcrun at: /Library/Developer/CommandLineTools/usr     &nbsp; &nbsp;   &nbsp; &nbsp; &nbsp; > &nbsp;xcode-select --install
+* python 3.7 or later and [pip package manager](https://pypi.org/project/pip/)
+* [Nordic mergehex](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fug_nrfutil%2FUG%2Fnrfutil%2Fnrfutil_intro.html) merge 2 or 3 (not 4) .hex files into one.
+* [Nordic nrfutil](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fug_nrfutil%2FUG%2Fnrfutil%2Fnrfutil_intro.html) Creates DeviceFirmwareUpdate zip file from hex
+* [Nordic nRFconnect](https://www.nordicsemi.com/Software-and-tools/Development-Tools/nRF-Connect-for-desktop) for your desktop or phone to upload DFU to the Ruuvi and a means to transfer the DFU zip file to you phone.
+
+* A computer or phone with a bluetooth radio to receive advertisments from the ruuvi. See [Dealing with the data](https://github.com/ruuvi/ruuvitag_fw/wiki/Dealing-with-the-data)
+
+### Suggested 
+
+* [Ruuvi Dev kit board](https://shop.ruuvi.com/product/devkit/) and a USB power & data cable.
+* [Nordic nrfjprog](https://www.nordicsemi.com/Software-and-tools/Development-Tools/nRF-Command-Line-Tools) Erase, write and read from Nordic nRF5 System On Chip
+ * [Segger j-link](https://www.segger.com/downloads/jlink) package incluing j-link and JLinkRTTClient
+ 
+### Optional
+* [Ceedling](http://www.throwtheswitch.org/ceedling) Needs Ruby and gcc
+* [PVS-Studio Analyzer](https://www.viva64.com/en/pvs-studio/) Purchase
```

Comment:
PVS-Studio gives free licenses for open source projects hosted on GitHub, GitLab or BitBucket: https://www.viva64.com/en/b/0600/

Reply:
Please do make the changes. I just got tired of fighting with GitHub formatting :-(

Reply:
ok